python/lsst/jointcal/jointcal.py

```diff
@@ -479,24 +479,30 @@ class JointcalConfig(pipeBase.PipelineTaskConfig,
     writeInitMatrix = pexConfig.Field(
         dtype=bool,
         doc=("Write the pre/post-initialization Hessian and gradient to text files, for debugging. "
-             "The output files will be of the form 'astrometry_preinit-mat.txt', in the current directory. "
+             "Output files will be written to `config.debugOutputPath` and will "
+             "be of the form 'astrometry_[pre|post]init-TRACT-FILTER-mat.txt'. "
```

Comment:
For this and the following few config parameters, do you want to clarify that the output will be [astro|photo]metry....txt, or is this part of the photometry capability of jointcal being deprecated? I see that for writeInitialModel you are moving away from mentioning the photometry in the documentation.

Reply:
Partly that, and partly that trying to specify it in such a manner resulted in the (already unwieldy) filename becoming even more unwieldy. I think "of the form" hopefully is clear enough for a user to get it?

Reply:
Sounds good to me.